build/erpnext-nginx/Dockerfile

```diff
@@ -1,9 +1,14 @@
-ARG NODE_IMAGE_TAG=12-prod
+ARG NODE_IMAGE_TAG=12-buster-slim
```

Comment:
this will need to be changed in [`travis.py`](https://github.com/frappe/frappe_docker/blob/develop/travis.py) as well